### ISSUE TITLE
Checkbox components have no :focus state

### ISSUE THREAD

**Issue by rschamp** [LLK/scratch-www#628]
For accessibility, there should be visual feedback when a checkbox is focused on from the keyboard.

**Comment by Tymewalk** [comment]
@rschamp Maybe a small "halo glow" around the checkbox? Most other sites I've seen use that.

Hi, testing to see what GH does:

- [ ] Hey!
- [ ] Ooh, checkboxes!
- [ ] Wow, testing!

**Comment by mewtaylor** [comment]
Fixed by https://github.com/LLK/scratch-www/pull/664
